### ISSUE TITLE
clear within `replot` is not using the correct window number

### ISSUE THREAD

**Issue by esternin** [esternin/eXtrema#54]
```
@sample_graphs
clear\-replot
replot 5
replot 6
```
The last command, `replot 6` **erases** window 5 (top left quadrant) before re-drawing window 6 (bottom-left).
```
replot 7
replot 8
```
Here `replot 8` erases 7 before refreshing 8 (not perfect, but that's the nature of what's in 8, a separate issue).

https://github.com/esternin/eXtrema/issues/53 ("replot does not clear") may be related to this. It's possible that `replot` does clear, but the wrong window.

**Comment by esternin** [comment]
The same behaviour under wx 3.2.1

**Comment by esternin** [comment]
A stupid and obvious typo, fixed by [3b9c58a](https://github.com/esternin/eXtrema/commit/3b9c58a9dea44c359dcd82f009db4145137ae03d)
